src/proto/beats.proto

```diff
@@ -11,17 +11,36 @@ enum Source {
   USER = 2;
 }
 
-enum Type {
-  BEAT = 0;
-  BAR = 1;
-  PHRASE = 2;
+enum SectionType {
+  INTRO = 0;
+  INNER = 1;
+  OUTRO = 2;
```

Comment:
```suggestion
  INTRO = 0;
  OUTRO = 1;
  OTHER = 2;
```

The user can put the intro or outro somewhere that is not the very beginning or end of the track, so "inner" is somewhat misleading.